backend/capellacollab/settings/modelsources/t4c/models.py

```diff
@@ -2,15 +2,32 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import enum
+import json
+import logging
 import typing as t
 
+import pydantic
+import requests
 from pydantic import BaseModel
+from requests.exceptions import RequestException
 from sqlalchemy import CheckConstraint, Column, ForeignKey, Integer, String
 from sqlalchemy.orm import relationship
 
-# required for sqlalchemy
-import capellacollab.settings.modelsources.t4c.models
 from capellacollab.core.database import Base
+from capellacollab.core.models import ResponseModel
+
+log = logging.getLogger(__name__)
+
+
+def validate_rest_api_url(value: t.Optional[str]):
+    if value:
+        try:
+            requests.Request("GET", value).prepare()
+        except RequestException:
+            log.info("REST API Validation failed", exc_info=True)
+            raise ValueError(
+                "The provided TeamForCapella REST API is not valid."
+            )
```

Comment:
The validator needs to return the value.

Reply:
Good catch! I was too focused trying out that the validation does work, but never tried a valid string :D